video-snippets/simple-over-search-bar.html

```diff
@@ -0,0 +1,217 @@
+    margin: 0 auto 20px;
+    border-radius: 50%;
+    background: #0C99D5;
+    border: 3px solid #B2E5FB;
+    height: 94px;
+    width: 94px;
+    transition: all 0.2s ease;
+    font-family: fontello-{{ snippet_id }};
+}
+
+#logo-replacement-{{ snippet_id }} .replay::before {
+    content: '\e800';  /* Replay icon */
+}
+
+#logo-replacement-{{ snippet_id }} .play::before {
+    content: '\e801';  /* Play icon */
+}
+</style>
+
+<div id="simple-video-snippet-{{ snippet_id }}" class="snippet">
+  <div id="logo-replacement-{{ snippet_id }}">
+    <div class="video-container">
+      <div class="video-overlay">
+        <button class="button replay"></button>
+        <button class="button play show"></button>
+      </div>
+      <video poster="{{ video_poster }}" src="{{ video_url }}" />
+    </div>
+  </div>
+  <div>
+    <img class="icon" src="{{ icon }}" />
+    <p>{{ message|safe }}</p>
+  </div>
+</div>
+
+<script>
+//<![CDATA[
+(function() {
+    'strict';
+
+    var videoVisible = false;
+
+    var snippet = document.getElementById('simple-video-snippet-{{ snippet_id }}');
+    var logoContainer = document.getElementById('brandLogo');
+    var topContainer = document.getElementById("topSection");
+    var message = snippet.querySelector('.message');
+
+    var logoReplacement = document.getElementById('logo-replacement-{{ snippet_id }}');
+    var video = logoReplacement.querySelector('video');
+    var replay = logoReplacement.querySelector('.replay');
+
+    snippet.addEventListener('show_snippet', function(e) {
+        // Fade out logo and insert animation.
+        logoContainer.style.opacity = 1;
+        logoContainer.style.transitionProperty = 'opacity';
+        logoContainer.style.transitionDuration = '400ms';
+        logoContainer.style.opacity = 0;
+        topContainer.insertBefore(logoReplacement, topContainer.firstChild);
+
+        // Once the logo fades out, show the video.
+        logoContainer.addEventListener('transitionend', function() {
+            topContainer.removeChild(logoContainer);
+            logoReplacement.classList.add('show');
+            videoVisible = true;
+        });
+    });
+
+    video.addEventListener('click', function() {
+        logoReplacement.querySelector('.play').classList.remove('show');
+        video.play();
+        video.controls = 'controls';
+        video.classList.add('played');
+    });
+
+    // Track when the video is first played.
+    var hasPlayed = false;
+    video.addEventListener('playing', function() {
+        if (!hasPlayed) {
+            hasPlayed = true;
+            sendMetric('simple-video-play');
+        }
+
+        replay.classList.remove('show');
+    });
+
+    // Track progress through the video.
+    var lastProgress = 0;
+    var thresholds = [0.25, 0.5, 0.75, 1];
+    video.addEventListener('timeupdate', function() {
+        var currentProgress = video.currentTime / video.duration;
+        if (currentProgress > lastProgress && !video.paused) {
```

Comment:
So if video has ended, `currentProgress==1` and `video.paused==True` therefore you don't ever send `simple-video-threshold-1`, correct? Is this why you added line 206?